timbuctoo-instancev4/src/main/java/nl/knaw/huygens/timbuctoo/core/RdfImportSession.java

```diff
@@ -34,6 +35,11 @@ public static RdfImportSession cleanImportSession(String vreName, DataStoreOpera
     return cleanImportSession(vreName, dataStoreOperations, errorReporter, new PropertyFactory(errorReporter));
   }
 
+  public static RdfImportSession cleanImportSession(String vreName, DataStoreOperations dataStoreOperations,
```

Comment:
You should use a test-stub `RdfImportSessionStubs.rdfImportSessionWithErrorReporter()` that doesn't pollute the class as much. 

The other two cleanImportSessions are the ones that are minimally needed. A package private one that gives full access for testing and the minimal public one for production use.